src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -226,18 +221,45 @@ private void setUserInfoInThreadContext(User user) {
         }
     }
 
-    public PrivilegesEvaluatorResponse evaluate(
-        final User user,
-        String action0,
-        final ActionRequest request,
-        Task task,
-        final Set<String> injectedRoles
-    ) {
+    public PrivilegesEvaluationContext createContext(User user, String action0, ActionRequest request, Task task, Set<String> injectedRoles)
+        throws PrivilegesEvaluatorResponse.NotAllowedException {
+        if (!isInitialized()) {
```

Comment:
How are we hitting a scenario where the plugin is not initialized but we are calling createContext for the evaluator? It seems this should not arise right? If we can reach this state doesn't that mean something is wrong.

Reply:
"Not initialized" means here "the configuration roles.yml, rolesmapping.yml, and actiongroups.yml" has not yet been installed on the cluster. That's a quite common case for a freshly installed cluster.